File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// BluefinDecryptxP2pe SDK exists test

require_once __DIR__ . '/../bluefindecryptxp2pe_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = BluefinDecryptxP2peSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# BluefinDecryptxP2pe SDK exists test

require "minitest/autorun"
require_relative "../BluefinDecryptxP2pe_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BluefinDecryptxP2peSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
